treescript/src/treescript/merges.py

```diff
@@ -44,22 +59,35 @@ def touch_clobber_file(repo_path):
 async def apply_rebranding(config, repo_path, merge_config):
     """Apply changes to repo required for merge/rebranding."""
     log.info("Rebranding %s to %s", merge_config.get("from_branch"), merge_config.get("to_branch"))
-    if merge_config.get("version_files"):
-        current_version = get_version("browser/config/version.txt", repo_path)
-        next_version = f"{current_version.major_number}.{current_version.minor_number}"
 
+    version = get_version("browser/config/version.txt", repo_path)
+    current_major_version = version.major_number
+    if merge_config.get("incr_major_version", False):
+        version = version.bump("major_number")
```

Comment:
I'm glad `mozilla-version` is useful in this case!

Reply:
It is, thank you!